test/30_debate/clientfunctions.js

```diff
@@ -262,7 +262,7 @@ describe("Debate client functions", () => {
         it('should return one suggestion', async () => {
             await new Promise(resolve => {
                 client.emit('suggestQuestion', 'This is my suggestion', res => {
-                    res.should.equal(true);
+                    res.should.not.equal(false);
```

Comment:
Why are you changing `res.should.equal(true)` to `res.should.not.equal(false)` ? The first is more clear and better to my eyes for this reason.

Reply:
suggestQuestion was modified to return false on error and the suggestion id on success (it is a necessary modification for the client app). 
`res.should.not.equal(false)` allows me to verify that there is no error, rather than checking for a particular id.